src/pages/get-connected/index.tsx

```diff
@@ -17,6 +18,8 @@ export default function GetConnected() {
 
   const [name, setName] = useState("");
   const [phoneNumber, setPhoneNumber] = useState("");
+  const [isChristian, setIsChristian] = useState("Yes");
+
   const [location, setLocation] = useState(cgLocations[0]!.value);
```

Comment:
avoid using `!` to force `non-null assertion`, instead use an `useEffect` to check whether `cgLocations` is populated and its length is more than 1, then then set the location.